jest.config.js

```diff
@@ -0,0 +1,18 @@
+const config = {
+  coverageDirectory: '/coverage',
+  coveragePathIgnorePatterns: ['/node_modules'],
+  coverageThreshold: {
+    global: {
+      branches: 95,
```

Comment:
Incidentally, after running `npx jest --coverage` locally I notice that I am not even meeting my own suggestion for this. I will take a look and see what I can do to improve branch coverage.

Reply:
Some things added in **Add jest** that would be ideally added correctly there instead of fixed here.